src/cpp/shared/gui/msg.cpp

```diff
@@ -31,3 +35,13 @@ void GUIBroadcastMessage(heroWindow* hwnd, int f, int c, void* p) {
 	evt.payload = p;
 	hwnd->BroadcastMessage(evt);
 }
+
+int GUIGetDropdownSelection(heroWindow* hwnd, void* p) {
+  tag_message evt;
+  evt.yCoordOrFieldID = 100;
```

Comment:
I'm actually not sure how you determined that, but I'll try to look for defined constants again.